pi4j-core/src/main/java/com/pi4j/io/gpio/digital/impl/DefaultDigitalOutputConfigBuilder.java

```diff
@@ -63,7 +63,7 @@ public static DigitalOutputConfigBuilder newInstance(Context context) {
      */
     @Override
     public DigitalOutputConfigBuilder address(Integer address) {
-        this.properties.put(GpioConfig.ADDRESS_KEY, String.valueOf(address));
```

Comment:
This looks odd... Is this intended? If so, why / what does this do?

Reply:
all part of the existing builder code, so following the same approach to set the BUS value.